proxy/envoy/config.go

```diff
@@ -322,6 +324,20 @@ func buildOutboundRoutes(instances []*model.ServiceInstance, services []*model.S
 	return httpConfigs, tcpConfigs
 }
 
+func buildSslContext(hostname string, context *ProxyContext) *SslContext {
+	mesh := context.MeshConfig
+	serviceAccounts := context.Discovery.GetIstioServiceAccounts(hostname)
+	if mesh.AuthMode == "istio-tls" {
+		return &SslContext {
+			CertChainFile:          mesh.AuthConfigPath + "/cert_chain",
+			PrivateKeyFile:         mesh.AuthConfigPath + "/private_key",
```

Comment:
The directory for auth secrets should be dynamic. We might need to create a holder directory for each restart epoch and store secrets inside that directory. Can be done in a follow-up PR.